src/cmd-osbuild

```diff
@@ -397,7 +397,12 @@ main() {
     fi
 
     outdir=$(mktemp -p "${tmp_builddir}" -d)
-    runvm_with_cache -- /usr/lib/coreos-assembler/runvm-osbuild                                         \
+    if has_privileges && [ "${COSA_NO_KVM:-}" == "1" ]; then
+        cmd="env" # run outside of supermin if we are root already
```

Comment:
I wonder if this is worth more commenting here to say this is mostly for the RISC-V case currently until `/dev/kvm` is more commonplace there.

Reply:
Will do in a followup.